apps/web/Dockerfile

```
@@ -1,21 +1,55 @@
+ENV NEXT_TELEMETRY_DISABLED=1
 ENV NODE_ENV=production
 
-# Copy contents of standalone to (app/web{.env, .next, server.js}, node_modules, package.json)
-COPY .next/standalone ./
+# Build the application from web directory
+WORKDIR /app/apps/web
+RUN bun run build
+RUN bash scripts/handle-nextjs-standalone-build.sh
+
+# Production image
+FROM oven/bun AS runner
```

Hadolint (2.12.0)

[warning] Always tag the version of an image explicitly

(DL3006)